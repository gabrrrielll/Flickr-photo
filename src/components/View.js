import React from "react";

const View = (props) => {
    return props.state.items.map((item, index) => (
        <div className="item" key={index}>
            <div className="title">{item.title ? item.title : "Empty title"}</div>
            <div className="content">
                <img src={"https://farm" + item.farm + ".staticflickr.com/" + item.server + "/" + item.id + "_" + item.secret + "_q.jpg"} alt={item.title} />
            </div>
        </div>
    ));
};

export default View;
