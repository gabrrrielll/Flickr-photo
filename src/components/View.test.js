import React from "react";
import View from "./Search";
import { shallow } from "enzyme";

describe("View component", () => {
    it("should show the items", () => {
        const wrapper = shallow(<View />);

        expect(wrapper).toMatchSnapshot();
    });
});
