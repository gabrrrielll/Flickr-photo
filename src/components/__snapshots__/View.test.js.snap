// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`View component should show the items 1`] = `ShallowWrapper {}`;
